js/chocolatey-internal-url.js

```diff
@@ -1,21 +1,21 @@
 (function() {
-    $(function () {
-        $("code:contains('STEP 3 URL')").html(function (_, html) {
-            $(this).parent().next().remove();
-            return html.replace(/(STEP 3 URL)/g, '<span class="stepThreeUrl">$1</span>');
+    jQuery(function () {
+        jQuery("code:contains('STEP 3 URL')").html(function (_, html) {
+            jQuery(this).parent().next().remove();
+            return html.replace(/(STEP 3 URL)/g, '<span class="stepThreeUrl">jQuery1</span>');

```

Comment:
I am not sure if this was intended, was it?

Reply:
Good catch, I have changed this back to the `$1` and have testing that it is working.